README.md

```diff
@@ -30,6 +30,27 @@ Although the community repository isn't technically a "yarn workspace", it funct
 
 Whenever a new changeset is introduced, a fresh "Version packages ($workspace_name)" PR is produced. Merging a Version packages PR will trigger the release of all the plugins in the workspaces (provided changesets have been added), and also update the `CHANGELOG` files.
 
+To create a changeset, follow these steps:
```

Comment:
This is great but should it be here instead? https://github.com/backstage/community-plugins/blob/main/CONTRIBUTING.md#creating-changesets